zypp/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ FILE( GLOB_RECURSE POT_FILE_DEPENDS RELATIVE ${LIBZYPP_SOURCE_DIR} "*.h" "*.cc"
 SET( POT_FILE_DEPENDS ${POT_FILE_DEPENDS} PARENT_SCOPE )
 
 
-ADD_DEFINITIONS(-DLOCALEDIR="${CMAKE_INSTALL_PREFIX}/share/locale" -DTEXTDOMAIN="zypp" -DZYPP_DLL )
+ADD_DEFINITIONS(-DLOCALEDIR="${CMAKE_INSTALL_PREFIX}/share/locale" -DTEXTDOMAIN="zypp" -DZYPP_DLL -DBUILDING_LIBZYPP )
```

Comment:
`-DZYPP_DLL (APIConfig.h)` and `-DBUILDING_LIBZYPP (global.h)` serve the same purpose. Please merge.